frontend/src/App.test.tsx

```diff
@@ -74,7 +82,7 @@ test("data is fetched again after refresh interval", async () => {
   // to rerender after every fetch (just in case)
   view.rerender(<App />);
 
-  const new_temp = screen.getByText("100 °C", { exact: false });
+  const new_temp = await screen.findByText("100 °C", { exact: false });
```

Comment:
NIT: `temp` name here is not descriptive

Reply:
See above lol, renamed to `gpu_temp`